examples/src/main/java/io/dapr/examples/invoke/http/HelloWorldClient.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ */
+
+package io.dapr.examples.invoke.http;
+
+import io.dapr.client.DaprClient;
+import io.dapr.client.DaprClientBuilder;
+import io.dapr.client.domain.Verb;
+
+/**
+ * 1. Build and install jars:
+ * mvn clean install
+ * 2. Send messages to the server:
+ * dapr run --port 3006 -- mvn exec:java -pl=examples -Dexec.mainClass=io.dapr.examples.invoke.http.HelloWorldClient -Dexec.args="'message one' 'message two'"
+ */
+public class HelloWorldClient {
```

Comment:
This file and class should have a different name, maybe ServiceInvoker or ServiceInvokerExample.

Reply:
Will do.

Reply:
Renamed it to InvokeClient.